[Container app] az containerapp env java-component: Support new configuration parameters on java component (#8320)
Azure/azure-cli-extensions: src/containerapp/HISTORY.rst

upcoming
++++++
* 'az containerapp debug': Open an SSH-like interactive shell within a container app debug console.
* 'az containerapp env java-component': Support more flexible configuration updates with new parameters --set-configurations, --replace-configurations, --remove-configurations and --remove-all-configurations.

> * 'az containerapp env java-component gateway-for-spring': Support --bind --unbind in create/update.

Reviewer: ```suggestion
* 'az containerapp env java-component gateway-for-spring create/update': Support `--bind` and `--unbind`
```

Author: updated as the suggestions
